sklearn/linear_model/tests/test_base.py

```diff
@@ -94,6 +94,18 @@ def test_linear_regression_sample_weights():
                 assert_almost_equal(inter1, coefs2[0])
 
 
+def test_raises_value_error_if_positive_and_sparse():
+    error_msg = ('A sparse matrix was passed, '
+                 'but dense data is required.')
+    # X, y must not be sparse if positive == True
```

Comment:
this seems to only test X, not y?

Reply:
Yes, my understanding was that the sparsity of the input feature matrix should be tested, not the target. The correspondent error is generated when X is sparse: am I wrong?

Reply:
The comment seems to imply that both X and y are tested so I'm a bit confused.

Reply:
You are right, sorry, I'm changing the comment.